src/test/java/com/debatetimer/service/member/MemberServiceTest.java

```diff
@@ -27,10 +28,10 @@ class CreateMember {
 
             MemberCreateResponse actual = memberService.createMember(request);
 
-            long memberCount = memberRepository.count();
+            Optional<Member> foundMember = memberRepository.findById(actual.id());
             assertAll(
                     () -> assertThat(actual.nickname()).isEqualTo(request.nickname()),
-                    () -> assertThat(memberCount).isEqualTo(1L)
+                    () -> assertThat(foundMember).isNotEmpty()
```

Comment:
존재하는지를 검증하는 isPresent 검증부가 있었던 것 같은데 조금 더 의미가 명확해질 것 같아요.

```suggestion
                    () -> assertThat(foundMember).isPresent()
```

Reply:
[반영 완료]

`MemberServiceTest.CreateMember`, `ParliamentaryServiceTest.DeleteTable` 모두 수정했습니다~